tests/test-pull-corruption.sh

```diff
@@ -55,7 +55,16 @@ do_corrupt_pull_test() {
 
 # FIXME - ignore errors here since gjs in RHEL7 has the final
 # unrooting bug
-gjs $(dirname $0)/corrupt-repo-ref.js ${repopath} main || true
+
+# super hacky fix to support both installed and uninstalled tests. we'll
+# probably want to just enable it for one of them.
+if [ -f /lib64/libostree-1.so ]; then
```

Comment:
I think a better approach to this is to set `LD_LIBRARY_PATH` for the uninstalled tests. I'm about to submit a branch that fixes that, among other things.